apps/redi-connect/src/routes/routes__logged-out.tsx

```diff
@@ -84,7 +84,7 @@ export const routes__loggedOut: RouteDefinition[] = [
   {
     path: '/front/login/entra-redirect',
     component: () => {
-      window.location.href = `${NEST_API_URL}/auth/entra-id`
+      window.location.href = `${NEST_API_URL}/api/auth/entra-id`
```

Comment:
@stuartmclean What's the reason for this change? Is it a common practice to set a URL on the browser to what seems to be an API URL?

Reply:
This is a bit of a hack, I agree, but it doesn't actually display anything to the user and it's the only way I could make this work without setting up a whole different root url just for this one endpoint. Happy to hear other suggestions though. 😎